package.json

```diff
@@ -40,11 +41,11 @@
     "react-use": "^15.3.3"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.1.1-alpha.18",
-    "@backstage/dev-utils": "^0.1.1-alpha.18",
-    "@backstage/test-utils": "^0.1.1-alpha.18",
     "@rollup/plugin-commonjs": "^14.0.0",
     "@rollup/plugin-node-resolve": "^8.4.0",
+    "@backstage/cli": "^0.1.1-alpha.21",
```

Comment:
those devDependencies are not sorted alphabetically

Reply:
fixed. Added also other rollup dependencies for the rollup build support.